docs/2018-10-24-Configuration-Splitting-Simplification.md

```diff
@@ -0,0 +1,88 @@
+Configuration Splitting Simplification
+======================================
+
+Background
+----------
+
+When doing one-path reachability proofs and applying an axiom,
+`∀ Z . α(Z) → ● β(Z)`, where `●` is the strong-next symbol (aka 'next'),
+to the following reachability goal
+```
+∀ X . φ(X) → ◆ ∃ Y . ψ(X, Y)
+```
+the result will look something like
+```
+￼(∀ X . Φ'(X) → ◆ ∃ Y . ψ(X, Y))
+￼∧
+￼(∀ X . Φ(X) ∧ (¬ ∃ Z. α(Z)) → ● ◆ ∃ Y . ψ(X, Y))
+```
+where `Φ` is either `φ` or `φ` and-ed with something else.
+See `one-path-reachability-proofs.md` for details.
+
+Problem
+-------
+
+We want to rewrite `φ(X) ∧ (¬ ∃ Z. α(Z))` part of the pattern above
+to something more manageable, preferably something that does not use `not`
+and `exists`, except in cases where it can be handled by an SMT solver.
+
+We will assume that
+1. `φ(X) = t(X) ∧ p(X)` where `p(X)` is a predicate, and `t(X)` is a
```

Comment:
Confusing because `=` is also a matching logic operator. Use "`φ(X)` is `t(X) ∧ p(X)`" (preferred), or perhaps another operator like `≡`.

Reply:
Wait, are you assuming `φ(X)` is given in that form, or just that it's provably equal to something in that form? (the later doesn't sound good for an algorithm, unless you require `t` and `p` as inputs).